forloop_modules/function_handlers/webscraping_handlers.py

```diff
@@ -301,6 +301,23 @@ def direct_execute(self, url, take_screenshot):
                         suh.webscraping_client.set_browser_meta_data(browser_meta_data)
 
             suh.webscraping_client.load_website(url)
+            
+            # get headers and cookies
+            headers = suh.webscraping_client.get_browser_headers()
+            cookies = suh.webscraping_client.get_browser_cookies()
+            requests = suh.webscraping_client.get_browser_requests()
+            
+            # get active pipeline uid
+            pipeline_uid=aet.active_pipeline_uid
+            
+            redis_key = f'pipeline:{pipeline_uid}:browser_headers'
```

Comment:
Use `RedisConfig` please. It should work similarly to e.g. `LAST_ACTIVE_SCRIPT_KEY_TEMPLATE`, i.e. you use `redis_config.LAST_ACTIVE_SCRIPT_KEY_TEMPLATE.format(...)`.

Take a look how that is used and replicate it for your usecase.

Reply:
So your usecase might be something like:
`redis_key = redis_config.BROWSER_HEADERS_KEY_TEMPLATE.format(pipeline_uid=pipeline_uid)`

Reply:
Maybe think about putting headers, cookies and requests into one JSON to Redis under one key (something like `BROWSER_OBJECTS_KEY_TEMPLATE`). Don't know if it's better